src/integrationTest/java/com/mongodb/hibernate/BasicCRUDTests.java

```diff
@@ -41,10 +42,12 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
-@SessionFactory(exportSchema = false)
-@DomainModel(annotatedClasses = {BasicCrudTests.Book.class, BasicCrudTests.BookWithEmbeddedField.class})
-class BasicCrudTests implements SessionFactoryScopeAware {
+@SessionFactory(exportSchema = false, useCollectingStatementInspector = true)
+@DomainModel(annotatedClasses = {BasicCRUDTests.Book.class, BasicCRUDTests.BookWithEmbeddedField.class})
+class BasicCRUDTests implements SessionFactoryScopeAware {
```

Comment:
1. You renamed this class to `BasicCrudIntegrationTests` in https://github.com/mongodb/mongo-hibernate/pull/57. What's the point of renaming it to `BasicCRUDTests` here?
2. We capitalize only the first letter, if needed, in an acronym in names of program elements: `UuidCodec` (the acronym is "UUID"), `AutomaticPojoCodec` (the acronym is "POJO"), JsonMode` (the acronym is "JSON").

So let's undo the change, and merge `main` into this branch once https://github.com/mongodb/mongo-hibernate/pull/57 is merged into `main`.